src/context/ActionSidebarContext/ActionSidebarContextProvider.tsx

```diff
@@ -51,7 +53,12 @@ const ActionSidebarContextProvider: FC<PropsWithChildren> = ({ children }) => {
 
   actionSidebarUseRegisterOnBeforeCloseCallback((element) => {
     const isClickedInside = isElementInsideModalOrPortal(element);
-    if (!isClickedInside) {
+    const navigationWrapper = document.querySelector('.modal-blur-navigation');
+
+    if (
+      !isClickedInside ||
+      (navigationWrapper && navigationWrapper.contains(element) && !isTablet)
```

Comment:
Can we please refactor this part `navigationWrapper && navigationWrapper.contains(element)` into an utils function?
```
const isChildOf = (parent, child) => parent && parent.contains(child);
```
and use it here `(isChildOf(navigationWrapper, element) && !isTablet)`